test/src/main/java/org/apache/accumulo/test/ScanServerIT.java

```diff
@@ -156,19 +148,10 @@ public void testScanOfflineTable() throws Exception {
       String tableName = getUniqueNames(1)[0];
 
       client.tableOperations().create(tableName);
-
```

Comment:
By removing this code block, and the code block below, you are testing that the code path for scanning using the TabletServer will fail for an offline table. The code that's here is testing that the code path for using the ScanServer with an offline table will fail.

Reply:
I see. I was trying to put the minimal amount of code into the assertThrows(), is what was there the minimum needed?

Reply:
The TableOfflineException is thrown when the scanner is created

Reply:
Short answer: yes.

If you don't insert data and flush it, then there will be no files in HDFS for the ScanServer to scan over. All the data will remain in the TabletServer in-memory buffer. If the client does not call `setConsistencyLevel(ConsistencyLevel.EVENTUAL)` on the Scanner, then the scan will go to the TabletServer, not the ScanServer.

Reply:
I see, thanks for clearing that up.

> If the client does not call `setConsistencyLevel(ConsistencyLevel.EVENTUAL)` on the Scanner, then the scan will go to the TabletServer, not the ScanServer.

It looks like the exception it throw before this can even be called though

Reply:
Without that method call, the code is going down the code path for scanning the Tablet Server, which is not allowed to scan offline tables either.

Reply:
oh, I see what you are saying...

Reply:
I removed these in the latest commit.